src/components/ContinuationIn/ContinuationAuthorization.vue

```diff
@@ -230,17 +232,20 @@ export default class ExtraproRegistration extends Mixins(DocumentMixin) {
 
   get authorizationDateRules (): Array<VuetifyRuleFunction> {
     return [
-      (v) => !!v || 'Authorization Date is required',
-      () => (this.authorization.date <= this.getCurrentDate) ||
-        'Authorization Date cannot be in the future'
+      () => !!this.authorization.date ||
+        'Authorization Date is required',
+      () => {
+        return (this.authorization.date <= this.getCurrentDate) ||
+          'Authorization Date cannot be in the future'
+      }
```

Comment:
Just a quick note that I used the local variable reference instead of `(v)` for the rule functions above because the latter is a formatted string like "May 14, 2024", which I cannot use for comparisons, while the former is a date string like "2024-05-14". This is a peculiarity of the Vuetify date picker and/or our wrapper around it.